src/pages/Minji/Main/임시보관.js

```diff
@@ -31,8 +35,29 @@ const Main = () => {
       setFiles(selectedFile.name);
     }
   };
+  const signiup = () =>
+    fetch('http://10.58.52.85:8000/signUp', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json;charset=utf-8',
+      },
+      body: JSON.stringify({
+        email: email,
+        password: password,
+      }),
```

Comment:
객체의 key와, value로 사용할 변수명이 같다면  아래처럼 축약해서 사용할  수 ㅇ있습니다
```suggestion
      body: JSON.stringify({
        email,
        password,
      }),
```